libparsec/crates/platform_mountpoint/src/lib.rs

```diff
@@ -0,0 +1,100 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+mod error;
+mod memfs;
+#[cfg(target_os = "windows")]
+mod windows;
+
+use chrono::{DateTime, Utc};
+use std::{collections::HashMap, path::Path};
+#[cfg(target_os = "windows")]
+pub use windows::mount;
+
+use libparsec_types::{EntryID, EntryName, FileDescriptor};
+
+pub(crate) use error::{MountpointError, MountpointResult};
+pub use memfs::MountpointManager;
+
+// TODO: remove me once ParsecEntryInfo is used
+#[allow(dead_code)]
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub(crate) enum EntryInfoType {
+    Dir,
+    File,
+    ParsecEntryInfo,
```

Comment:
Can you add a comment to explain what is this ?

Reply:
Done